### PR TITLE
Removes lr from default_optimizer_params_registry

### LINKED ISSUE #1621: Inconsistent learning rates shown in output when specifying learning rate in config
When specifying the learning_rate parameter in the training section of the config, `description.json` shows a different default learning rate in the optimizer section:

config.yaml:
Screen Shot 2021-12-21 at 4 46 42 PM

description.json:
Screen Shot 2021-12-21 at 4 47 04 PM

Expected behavior is that there is one learning rate, and if it is specified in the config it should match that.

### PR DESCRIPTION
**Pull request by dantreiman**
Fixes #1621

Removes `training.optimizer.lr` defaults from `default_optimizer_params_registry`.

Use `training.learning_rate` as default value for`training.optimizer.lr` instead.

`training.learning_rate` will always be present, either from config or `default_training_params`
```
default_training_params = {
    "learning_rate": 0.001,
```

### CODE REVIEW

**Review by justinxzhao — APPROVED** [review]
LGTM
